instantshare.spec

```diff
@@ -0,0 +1,51 @@
+# -*- mode: python -*-
+
+block_cipher = None
+
+hidden_imports = ['cli.audio', 
+                  'cli.file', 
+                  'cli.gui', 
+                  'cli.screen', 
+                  'cli.text', 
+                  'cli.video',
+                  'screenshot.snippingtool',
+                  'screenshot.windows_tk',
+                  'screenshot.gnome_screenshot',
+                  'storage.dropbox',
+                  'storage.googledrive',
+                  'storage.imgur',
+                  'storage.owncloud',
+                  'storage.pastebin',
+                  'storage.sftp',
+                  'storage.twitter_intent']
+
+added_files = [('src/res', 'res')]
+
+a = Analysis(['src\\instantshare'],
+             pathex=['C:\\Developer\\instantshare'],
```

Comment:
Completely overlooked that the file contains system specific file paths. Will look into this.